### PR TITLE
Documentation − fixes

### PR DESCRIPTION
**Pull request by woecifaun**

### CODE REVIEW

**Comment by woecifaun** [comment]
You also have links in 404 in https://github.com/Phpillip/phpillip/blob/master/doc/controller/content.md (show/list/page)

**Comment by woecifaun** [comment]
And I'm not sure but I believe something is not square at line 18 of https://github.com/Phpillip/phpillip/blob/master/doc/content/retrieving-content.md

File: doc/more/about.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # About Phpillip
 
 Hi, I'm [Thomas Jarrand](https://github.com/tom32i).
-I'm a web developer that use Symfony a lot, I created and maintain Phpillip.
+I'm a web developer using Symfony a lot, I created and maintain Phpillip.
 
 I first built it as a tool to create my [technical blog](http://thomas.jarrand.fr/blog/).
 
-It's not meant to be the best PHP Website Generator around: its first goal was to feet my needs.
```
Comment (Tom32i) [inline doc/more/about.md]: Oh gosh that on T_T

File: doc/more/contribution.md [hunk 2]
```diff
@@ -12,6 +12,6 @@ __Whether you're a developer or not, any contribution are welcome:__
 - Idea / suggestion
 - Sharing / Blog post
 
-If you want to contribute to the source code, please fill submit a pull request in Github.
+If you want to contribute to the source code, please fill free to submit a pull request in Github.
```
Comment (Tom32i) [inline doc/more/contribution.md]: *feel* T_T

**Comment by Tom32i** [comment]
This is great! Thank you so much for your time :)

**Pull request merged**